test/test_fat.rb

```diff
@@ -5,6 +5,40 @@
 class FatFileTest < Minitest::Test
   include Helpers
 
+  def test_empty_file
+    empty_file = tempfile_with_data("empty_file", "")
+
+    assert_raises MachO::NotAMachOError do
+      MachO::FatFile.new(empty_file)
+    end
+  ensure
+    empty_file.close
+    empty_file.unlink
```

Comment:
I agree. I could probably turn `tempfile_with_data` into a block, that would reduce the repetition and avoid confusion with the interaction between `Helpers` and each test class.